package-extension.sh

```
@@ -0,0 +1,45 @@
+cp output.css dist/
+cp settings.html dist/
+cp settings.js dist/
+
+# Icons
+mkdir -p dist/icons
+cp icons/icon128.png dist/icons/
+cp icons/icon1285.png dist/icons/
+
+# Create the ZIP file
+echo "Creating ZIP package..."
+cd dist
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)